src/Response/DirectPlusResponse.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Nexy\PayboxDirect\Response;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class DirectPlusResponse extends AbstractResponse
+{
+    /**
+     * @var string
+     */
+    private $subscriberRef;
+
+    /**
+     * @var string|false False if empty.
```

Comment:
Well, I decided to follow same logic is this: https://github.com/nexylan/paybox-direct/pull/32/files#diff-e06a2f8820ff95ef72cc9e316326ad9dR46

* `null` when the property is not returned by Paybox
* `false` when it's return but invalid (empty or `???` for country)

In this case, the bearer is **always** but can sometime be empty (on subscriber delete if i'm right.).